src/locales/fr/WCAG.json

```
@@ -0,0 +1,729 @@
+        },
+        "1.4.13": {
+            "TITLE": "Contenu au survol ou au focus ",
+            "DESCRIPTION": "Lorsque la réception puis le retrait du survol du pointeur ou du focus du clavier déclenche l’affichage puis le masquage d’un contenu additionnel, les éléments suivants sont vrais :",
+            "DETAILS": {
+                "1": {
+                    "DESCRIPTION": "Il existe un mécanisme permettant de masquer le contenu additionnel sans déplacer le pointeur ou le focus du clavier, à moins que le contenu additionnel ne communique une erreur de saisie ou ne masque ou ne remplace un autre contenu.",
+                    "TITLE": "Masquer"
+                },
+                "2": {
+                    "DESCRIPTION": "Si le survol du pointeur peut déclencher le contenu additionnel, alors le pointeur peut être déplacé sur le contenu additionnel sans que celui-ci disparaisse.",
+                    "TITLE": "Survoler"
+                },
+                "3": {
+                    "DESCRIPTION": "Le contenu additionnel reste visible jusqu’à ce que le survol ou le focus soit retiré, que l’utilisateur le masque ou que ses informations ne soient plus valables.",
+                    "TITLE": "Persister"
+                }
+            }
+        },
+        "2.1.1": {
+            "TITLE": "Clavier",
+            "DESCRIPTION": "Toutes les fonctionnalités du contenu sont utilisables à l’aide d’une interface clavier sans exiger un rythme de frappe propre à l’utilisateur, sauf lorsque la fonction sous-jacente nécessite une saisie qui dépend du tracé du mouvement effectué par l’utilisateur et pas seulement des points de départ et d’arrivée de ce tracé.",
+            "DETAILS": {}
+        },
+        "2.1.2": {
+            "TITLE": "Pas de piège au clavier",
+            "DESCRIPTION": "Si le focus du clavier peut être positionné sur un élément de la page à l’aide d’une interface clavier, réciproquement, il peut être déplacé hors de ce même composant simplement à l’aide d’une interface clavier et, si ce déplacement exige plus que l’utilisation d’une simple touche flèche ou tabulation ou toute autre méthode standard de sortie, l’utilisateur est informé de la méthode permettant de déplacer le focus hors de ce composant.",
+            "DETAILS": {}
+        },
+        "2.1.3": {
+            "TITLE": "Clavier (pas d’exception)",
+            "DESCRIPTION": "Toutes les fonctionnalités du contenu sont utilisables à l’aide d’une interface clavier sans exiger un rythme de frappe propre à l’utilisateur.",
+            "DETAILS": {}
+        },
+        "2.1.4": {
+            "TITLE": "Raccourcis clavier utilisant des caractères",
+            "DESCRIPTION": "Si un raccourci clavier est mis en œuvre dans du contenu en utilisant uniquement des caractères de type lettres (y compris les majuscules et les minuscules), des signes de ponctuation, des chiffres ou des symboles, alors au moins l’une des conditions suivantes est vraie :",
```

```suggestion
            "DESCRIPTION": "Si un raccourci clavier est implémenté dans du contenu en utilisant uniquement des caractères de type lettres (y compris les majuscules et les minuscules), des signes de ponctuation, des chiffres ou des symboles, alors au moins l’une des conditions suivantes est vraie :",
```

See https://www.w3.org/Translations/WCAG21-fr/#character-key-shortcuts
Please use the exact wording of the Authorized Translation